src/System.Reflection.DispatchProxy/src/System.Reflection.DispatchProxy.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
```

Comment:
Looking at the 4.7.1 package on nuget.org I see that it has a functional (non PNSE throwing) assembly for netcoreapp2.0. Don't we want/need a .NETCoreApp TFM anymore?

Reply:
When attempting to consume the nuget package in a console app targeting .NET 6.0, the nuget package gets ignored. We directly consume the one that comes inbox. See [my comment](https://github.com/dotnet/maintenance-packages/pull/34#issuecomment-1897355725).

So my understanding is that there's no point in targeting .NET 6.0 in maintenance-packages due to the above. Please correct me if I got this wrong, @ericstj.

Reply:
Right, we need to decide how we handle these cases.

For other packages that actually have a functional implementation on .NETCore we've proposed raising that implementation target framework to the minimum in support framework - `net6.0`.  For out-of-support frameworks they'd get whatever netstandard assembly was there (PNSE or whatever) and get the warning that their framework was out of support.

In this case the minimum in support framework would not get the package implementation at all due to conflict resolution - so there's really no point in building one and putting it in the package.

We do need to make sure the warning targets are in place though.

Reply:
> We do need to make sure the warning targets are in place though.

Agreed, having the targets in place - even without a .NETCoreApp TFM in the package - is important.

Reply:
Can you show me which targets you're referring to? Have we done this for the other packages already?

Reply:
This target: https://github.com/dotnet/maintenance-packages/blob/3a51338ad1494c6427d7c3fc308f3e6429ab7b57/eng/packaging.targets#L55-L60

Reply:
Okay I see. You're talking about the targets that already exist and I need to run `dotnet pack` as described in the most recent comment by Viktor. So I think I can resolve this specific conversation.